appsTransfer/src/main/java/org/kinecosystem/appstransfer/presenter/SenderServiceBinder.kt

```diff
@@ -80,39 +80,21 @@ class SenderServiceBinder(private val context: Context?) {
         }
     }
 
-    fun startSendKin(receiverAddress: String, senderAppName: String, amount: Int, memo: String, receiverPackage: String) {
+    fun startSendKin(receiverAddress: String, amount: Int, memo: String) {
```

Comment:
make sure you rebase your both the feature branch and this branch from master cause there are changes here ...